hypervisor/src/mshv/mod.rs

```diff
@@ -929,6 +932,22 @@ impl vm::Vm for MshvVm {
             )
             .map_err(|e| vm::HypervisorVmError::GetDirtyLog(e.into()))
     }
+    ///
+    /// Enable dirty page tracking
+    ///
+    fn enable_dirty_page_tracking(&self) -> vm::Result<()> {
```

Comment:
I don't think this API is needed, as  `self.fd.enable_dirty_page_tracking()` is better to be called directly from `start_dirty_log()`.